br-webui/views/camera.liquid

```diff
@@ -7,7 +7,7 @@
 <script src="js/bootstrap-slider.js"></script>
 
 <script>
-	var socket = io();
+	//var socket = io();
```

Comment:
Can be deleted.

Reply:
oh yeah.. I think i missed that one by mistake.